src/main/java/frc/trigon/robot/subsystems/swerve/SwerveModuleConstants.java

```diff
@@ -18,46 +23,54 @@ public class SwerveModuleConstants {
             STEER_MOTOR_NEUTRAL_MODE_VALUE = NeutralModeValue.Brake;
     private static final double
-            DRIVE_SLIP_CURRENT = RobotHardwareStats.isSimulation() ? 1000 : 100,
-            STEER_CURRENT_LIMIT = RobotHardwareStats.isSimulation() ? 1000 : 50;
+            DRIVE_SLIP_CURRENT = RobotHardwareStats.isSimulation() ? 1000 : 80,
+            STEER_CURRENT_LIMIT = RobotHardwareStats.isSimulation() ? 1000 : 30;
     private static final double
             STEER_MOTOR_P = RobotHardwareStats.isSimulation() ? 75 : 75,
             STEER_MOTOR_I = 0,
-            STEER_MOTOR_D = 0;
+            STEER_MOTOR_D = 0,
+            STEER_MOTOR_KV = 1,
+            STEER_MOTOR_KA = 1;
     private static final double
             DRIVE_MOTOR_P = RobotHardwareStats.isSimulation() ? 50 : 50,
             DRIVE_MOTOR_I = 0,
-            DRIVE_MOTOR_D = 0;
+            DRIVE_MOTOR_D = 0,
+            DRIVE_MOTOR_KS = RobotHardwareStats.isSimulation() ? 1 : 0,
+            DRIVE_MOTOR_KV = RobotHardwareStats.isSimulation() ? 1 : 0,
+            DRIVE_MOTOR_KA = RobotHardwareStats.isSimulation() ? 1 : 0;
     static final boolean ENABLE_FOC = true;
     static final TalonFXConfiguration
             DRIVE_MOTOR_CONFIGURATION = generateDriveConfiguration(),
             STEER_MOTOR_CONFIGURATION = generateSteerConfiguration();
     static final CANcoderConfiguration STEER_ENCODER_CONFIGURATION = generateSteerEncoderConfiguration();
 
-    private static final double
-            DRIVE_MOMENT_OF_INERTIA = 0.03,
-            STEER_MOMENT_OF_INERTIA = 0.003;
     private static final int
             DRIVE_MOTOR_AMOUNT = 1,
             STEER_MOTOR_AMOUNT = 1;
     private static final DCMotor
             DRIVE_MOTOR_GEARBOX = DCMotor.getKrakenX60Foc(DRIVE_MOTOR_AMOUNT),
             STEER_MOTOR_GEARBOX = DCMotor.getFalcon500Foc(STEER_MOTOR_AMOUNT);
 
-    static final double WHEEL_DIAMETER_METERS = RobotHardwareStats.isSimulation() ? 0.1016 : 0.049149 * 2;
+    static final SysIdRoutine.Config DRIVE_MOTOR_SYSID_CONFIG = new SysIdRoutine.Config(
+            Units.Volts.of(5).per(Units.Second),
+            Units.Volts.of(8),
+            Units.Second.of(1000)
+    );
+
+    static final double WHEEL_DIAMETER_METERS = RobotHardwareStats.isSimulation() ? 0.1016 : 0.049274 * 2;
     static final double VOLTAGE_COMPENSATION_SATURATION = 12;
 
     static SimpleMotorSimulation createDriveSimulation() {
-        return new SimpleMotorSimulation(DRIVE_MOTOR_GEARBOX, DRIVE_GEAR_RATIO, DRIVE_MOMENT_OF_INERTIA);
+        return new SimpleMotorSimulation(DRIVE_MOTOR_GEARBOX, DRIVE_GEAR_RATIO, DRIVE_MOTOR_KV, DRIVE_MOTOR_KA);
     }
 
     static SimpleMotorSimulation createSteerSimulation() {
-        return new SimpleMotorSimulation(STEER_MOTOR_GEARBOX, STEER_GEAR_RATIO, STEER_MOMENT_OF_INERTIA);
+        return new SimpleMotorSimulation(STEER_MOTOR_GEARBOX, STEER_GEAR_RATIO, STEER_MOTOR_KV, STEER_MOTOR_KA);
```

Comment:
using moi is better. this is a number that can vary because of torque control and stuff, and I'd rather one sim MOI value.

Reply:
The old constructor with moi doesn't exist anymore. There's only one constructor with ka and kv